commands/a_utility/pronouns.js

```diff
@@ -1,7 +1,9 @@
 const { Command } = require('@sapphire/framework');
-const { Interaction, MessageEmbed, PermissionFlagsBits } = require('discord.js');
+const { Interaction, MessageEmbed, PermissionFlagsBits, Guild, Message, MessageManager } = require('discord.js');
 const firebaseUtil = require('../../db/firebase/firebaseUtil');
 
+var emojis = ['1️⃣', '2️⃣', '3️⃣', '4️⃣'];
```

Comment:
small comment but should we change this to `let`?

Reply:
Valid - changed